espnet2/bin/spk_inference.py

```diff
@@ -137,10 +134,10 @@ def inference(
         format="%(asctime)s (%(module)s:%(lineno)d) %(levelname)s: %(message)s",
     )
 
-    if ngpu >= 1:
-        device = "cuda"
-    else:
-        device = "cpu"
+    # if ngpu >= 1:
+    #     device = "cuda"
+    # else:
+    #     device = "cpu"
```

Comment:
If this is being commented out because of not being used, 
I believe adding it into `speech2embedding_kwargs` should be better. 
@ftshijt how do you think?